### PR TITLE
Add Go mod compatibility

### PR DESCRIPTION
**Pull request by shoukoo**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
 
 .PHONY: shell
 shell:
-	docker run -it -v $${GOPATH}/bin:/root/bin golang:stretch bash
+	docker run -it -v $${PWD}:/root/bin golang:stretch bash
 
 .PHONY: install
 install:
-	go install -ldflags "-s -w"
```
Comment (alexlance) [inline Makefile]: You've taken away my ldflags :(

Comment (shoukoo) [reply Makefile]: oh :( , what are those flag for?

Comment (alexlance) [reply Makefile]: They keep the file size of the binary small.

**Comment by alexlance** [comment]
I added go mod compatibility straight into master branch. I will close this PR for now, but will probably grab the tests out of it at some point as they look like a good start.

**Pull request closed without merging**